terps/unp64/CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
-add_library(unp64 STATIC 6502emu.c exo_util.c log.c unp64.c
-    scanners/_Scanners.c scanners/1001card.c scanners/AbuzeCrunch.c
-    scanners/ActionPacker.c scanners/ByteBoiler.c scanners/Caution.c
-    scanners/CCS.c scanners/Cruel.c scanners/ECA.c scanners/Exomizer.c
-    scanners/Expert.c scanners/FinalSuperComp.c scanners/Intros.c
-    scanners/MasterCompressor.c scanners/Megabyte.c scanners/MrCross.c
-    scanners/MrZ.c scanners/PuCrunch.c scanners/Section8.c
-    scanners/TBCMultiComp.c scanners/TCScrunch.c scanners/XTC.c)
-
-c_standard(unp64 11)
+set (CMAKE_CXX_STANDARD 17)
+
+add_library(unp64 STATIC 6502_emu.cpp exo_util.cpp globals.cpp unp64.cpp
+    scanners/scanners.cpp scanners/abuze_crunch.cpp scanners/action_packer.cpp
+    scanners/byte_boiler.cpp scanners/caution.cpp scanners/ccs.cpp scanners/cruel.cpp 
+    scanners/eca.cpp scanners/exomizer.cpp scanners/expert.cpp scanners/final_super_comp.cpp
+    scanners/intros.cpp scanners/master_compressor.cpp scanners/mc_cracken_comp.cpp
+    scanners/megabyte.cpp scanners/mr_cross.cpp scanners/mr_z.cpp scanners/pu_crunch.cpp
+    scanners/section8.cpp scanners/tbc_multicomp.cpp scanners/tcs_crunch.cpp
+    scanners/time_cruncher.cpp scanners/xtc.cpp)
+
+target_include_directories(unp64 INTERFACE ${CMAKE_CURRENT_SOURCE_DIR} PRIVATE PARENT_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+
 warnings(unp64)
 
```

Comment:
And again, since you'll already be in the file, an extraneous newline which can be removed.